### ISSUE TITLE
x/pkgsite: Accessibility Color Contrast Problems

### ISSUE THREAD

**Issue by feketegy** [golang/go#51480]
Would it be possible to change the color contrast on the pkg.go.dev website so people with color blindness or other eye deficiencies could enjoy and use the website better?
 
I made a test with the a11y.com tool to see the contrast between various colors on the website and the colors highlighted in the screenshot were problematic, these colors are mainly used in links, titles, and code backgrounds.

### Screenshot
 
<img width="1550" alt="image" src="https://user-images.githubusercontent.com/489775/156806762-b1cd0a04-7e45-4097-a286-e6e123d40cf6.png">
 
 
 
### What did you do?
 
1. Go to: https://color.a11y.com/Contrast/
2. Test a pkg.go.dev internal page, I tested: https://pkg.go.dev/encoding/json

**Comment by beoran** [comment]
In stead of relying on a tool which only simulates color blindness, you should ask people who have color blindness, like myself. Since the go website uses white as the background color, and black or blue as the foreground, the contrast is sufficient for me. Unlike, say the python docs which use green or yellow backgrounds... 😛

Of course, color blindness is very personal, so I would like to hear the opinion of a few other color blind people. But I can tell you that most tools like a11y are unhelpful since they do not and cannot describe what people like me actually see. For me the Ho website is fine.

**Comment by feketegy** [comment]
@beoran Is there a tool that you would suggest that comes close to what you're experiencing and what the tool tells you?

I asked a couple of people with color impairments and they told me that the a11y is good.

**Comment by beoran** [comment]
@feketegy, That is the problem, I haven't found any particular tool that is correct for me, and I tried a dozen on so. As I said color blindness is very personal. I but the Go website is fine for me. 

I think it would be best to try to ask the opinions of several color blind people before making any changes. If you make changes based on just a tool in stead of people, you risk making it worse for some people as well.

**Comment by beoran** [comment]
Personally don't see too much problems here, but if we really must increase the contrast a selectable high conrast theme could be the solution. The code examples could be white on black as in a terminal. The links coukd be underlined so tgey stand out better no matter what the color.

**Comment by bkaws** [comment]
Hello, any updates on the [submitted PR](https://github.com/golang/go/issues/51480#issuecomment-2100754930) that fixes the color contrast accessibility issue?

**Comment by bkaws** [comment]
@jamalc still looking for an update on the [PR](https://github.com/golang/go/issues/51480#issuecomment-2100754930) we submitted to fix the color contrast issue for light mode based on the WCAG 2.0 standards. The issue was flagged during an accessibility audit and we are still looking for a resolution. If you are not going to accept the PR that fixes the issue, can you let us know so we can share with the auditor? Thank you.

**Comment by bkaws** [comment]
@dmitshur @mvdan hello, could you folks please let us know whether you'll accept the [PR](https://github.com/golang/go/issues/51480#issuecomment-2100754930) we opened for the pkg.go.dev documentation pages to be [WCAG 2.2 color contrast](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum) compliant? If you suggest a different approach, could you please let us know and we can revise the PR? I'm sure you appreciate that addressing accessibility issues for our customers is a top priority and we have not had any communication about this issue or the PR opened from the maintainers since 5/8. Thank you.
